tgrsite/templates/tgrsite/basecontent.html

```diff
@@ -58,6 +59,26 @@
         {% block navbar %}{% endblock %}
     </div>
 
+    <div class="modal fade bd-example-modal-lg" id="tutorial-modal" tabindex="-1" role="dialog" aria-labelledby="tutorial-modal" aria-hidden="true">
```

Comment:
This almost certainly doesn't want to be in basecontent (as that gets included in emails as well). You probably want to add a block here, and then either overwrite the block to add it to main if you want it on every page, or just on the specific pages you want the tutorial to appear.

Reply:
Shit, good catch. I want it to appear on every page, so I'll apply that fix. (Will reply when done.)